src/i18n/i18n-types.ts

```diff
@@ -1170,6 +1171,7 @@ export type TranslationFunctions = {
                 };
             };
         };
+        editSmartPlaylist: () => LocalizedString;
```

Comment:
Are you adding these manually? If you run `npm run typesafe-i18n`  in another terminal, it should auto-generate the types for you based on `index.ts`, and will hot-reload them as well.

Reply:
Yep, I did...